test/unittests/commands/excel2json/test_list_node.py

```diff
@@ -0,0 +1,174 @@
+import pandas as pd
+import pytest
+
+from dsp_tools.commands.excel2json.models.input_error import ListNodeProblem
+from dsp_tools.commands.excel2json.models.input_error import ListProblem
+from dsp_tools.commands.excel2json.models.list_node import ListNode
+from dsp_tools.commands.excel2json.models.list_node import ListRoot
+
+
+class TestListRoot:
+    def test_make_myself_comments(self) -> None:
+        sub_1 = ListNode(id_="Node_1", labels={"en": "Node_1_label_en"}, row_number=1)
```

Comment:
Here I did not use the `.create()` method because of typing when the function that is tested is called.